packages/react-components/src/mail.ts

```diff
@@ -34,3 +34,14 @@ export const mailToSupport = (overrides?: MailOptions): string =>
     subject: 'ASAP Hub: Tech support',
     ...overrides,
   });
+
+export const convertEmailListToMailToUrl = (list: string[]): URL => {
+  const href = list.reduce(
+    (a, e, i) => {
+      const suffix = i === list.length - 1 ? '' : ',';
+      return `${a}${e.split('@').map(encodeURIComponent).join('@')}${suffix}`;
+    },
+    '',
+  );
+  return new URL(`mailto:${href}`);
+};
```

Comment:
```suggestion

```

Reply:
This isn't required anymore but if it was it shouldn't be exported if nothing else uses it